frontend/packages/topology/src/behavior/withCreateConnector.tsx

```diff
@@ -137,14 +140,17 @@ const CreateConnectorWidget: React.FC<CreateConnectorWidgetProps> = observer((pr
   element.translateFromParent(startPoint);
   element.translateFromParent(endPoint);
 
+  const classes = classNames('topology-create-connector', {
+    'is-dragging': dragging,
+  });
```

Comment:
I think we should remove this altogether and set `pointer-events: none` on the handle so that we let whatever is underneath control the cursor.